sys-devel/clang/clang-4.0.1.ebuild

```diff
@@ -98,6 +98,10 @@ src_prepare() {
 	# (backport, temporary reverted upstream because of FreeBSD issues)
 	eapply "${FILESDIR}"/4.0.1/0001-Frontend-Correct-values-of-ATOMIC_-_LOCK_FREE-to-mat.patch
 
+	# add tblgen for cross compile
+	# https://bugs.gentoo.org/667094
+	"${FILESDIR}"/fix-tblgen.patch
```

Comment:
Except the fact this is a really old version of clang, didn't you miss an eapply?